templates/accuracy_notice_email.html

```diff
@@ -0,0 +1,31 @@
+<p>
+    [Why me?] You are receiving this notification because you are listed as an owner of the following ChromeStatus feature entry:
+</p>
+</br>
+<p>
+    <strong>{{feature_name}}</strong>
+</p> 
+</br>
+<p>
+    [Why now?] Your feature is slated to launch soon:
+</p>
+</br>
+<p>
+    <strong>m{{milestone}}</strong>
+</p>
+</br>
+<p>
+    [Background/motivation] Your feature entry is an important resource for
+    cross functional teams that help drive adoption of new features and
+    enterprise IT admins who might be affected by web platform changes.
+</p>
+</br>
+<p>
+    [Call to action] We need to know whether your plans are changing or staying
+    the same. <strong>Please click the link below to update and confirm key
+    fields of your feature entry.</strong>
+</p>
+</br>
+<b><a href="https://chromestatus.com/guide/verify_accuracy/{{id}}">
```

Comment:
Rather than use `https://chromestatus.com/` here, let's use settings.SITE_URL.  That way the staging emails will have links that work.

Also, I notice that in settings.py SITE_URL uses `http`, we should update that to `https`.

Reply:
Done!